### DISCUSSION TITLE
posts/2022/03/making-tailwind-jit-work-hugo-version-3-edition/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#2] [Announcements]
# posts/2022/03/making-tailwind-jit-work-hugo-version-3-edition/

A Hugo fix for Tailwind CSS v.3—with a surprise bonus.

https://www.brycewray.com/posts/2022/03/making-tailwind-jit-work-hugo-version-3-edition/

**Comment by ivanduka** [comment]
Excellent article, thanks!
There is a downside, though - the resources folder is polluted now with the useless junk and cannot be checked in to Git anymore. It is in Git for me because image resizing is a CPU-costly operation, and I prefer to store the computed results in Git to avoid recalculation.

**Reply by brycewray** [reply]
@ivanduka No argument. That’s a dicey issue, to be sure. Thanks for reading!
